FirebasePerformance/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Version 8.0.1
+* Add Firebase console URL logging when debug logging is enabled.
```

Comment:
Nit: Firebase performance logs now contain URL to the see the performance metrics on the Firebase Console.

Reply:
Done.